Gemfile

```
@@ -1,15 +1,15 @@
 source "https://rubygems.org"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.2.1"
+gem "rails", "~> 8.0.1"
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem "sprockets-rails"
 # Use sqlite3 as the database for Active Record
-gem "sqlite3", ">= 1.4"
+gem "sqlite3", "~> 2.5.0"
 # Use the Puma web server [https://github.com/puma/puma]
-gem "puma", ">= 5.0"
+gem "puma", ">= 6.5.0"
 # Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
-gem "importmap-rails"
+gem "importmap-rails", "~> 2.1.0"
```

You don't need to add the versions in to bump the gem, as the version is locked in Gemfile.lock. It's actually a lot less hassle to have virtually no versions defined in Gemfile - see GOV.UK Chat for instance: https://github.com/alphagov/govuk-chat/blob/main/Gemfile and relevant GOV.UK documentation: https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html#gemfile-organisation


```suggestion
gem "importmap-rails"
```

Note, after changing you'll need to run `bundle install` again